src/huntsman/pocs/scheduler/dispatch.py

```diff
@@ -1,11 +1,12 @@
 from astropy import units as u
 
+from panoptes.utils import current_time
```

Comment:
New version will enforce this so just changing now.

```suggestion
from panoptes.utils.time import current_time
```